### PR TITLE
feat: add feature for manual IP addresses

### PR DESCRIPTION
**Pull request by sumit0861**
## Pull Request Submission Checklist

Please confirm that you have done the following before requesting reviews:

- [x] I have confirmed that the PR type is appropriate for the change I am making according to the [Honest Pull Request and Commit Message Naming Conventions](https://www.notion.so/honestbank/Pull-Request-and-Commit-Message-Naming-Conventions-bd97f2cbb34c4c73b1ff3a3e384b850c).
- [x] I have typed an adequate description that explains **why** I am making this change.
- [x] I have installed and run standard pre-commit hooks that lints and validates my code.

### Description

* Alto needs static list of IPs from Honest to be able to allow list our traffic.
* We till now had auto NAT IP allocation.
* This change allows us to set static list of NAT IPs

- - -
This change is [Reviewable](https://reviewable.io/reviews/honestbank/terraform-gcp-gke/117)

### CODE REVIEW

**Review by cyberhck — APPROVED** [review]

File: modules/gcp-gke/inputs.tf [hunk 1]
```diff
@@ -178,3 +178,9 @@ variable "maintenance_policy_config" {
   description = "(OPTIONAL) A list of objects used to configure maintenance policy "
   default     = []
 }
+
+variable "nat_ip_addresses" {
```
Comment (sunilhonest) [inline modules/gcp-gke/inputs.tf:182]: `nat_ip_addresses` should be declared as input variable in `terraform-gcp-gke` module too.

**Review by sunilhonest — CHANGES_REQUESTED** [review]

File: modules/gcp-gke/inputs.tf (hunk 1 above)
Comment (cyberhck) [reply modules/gcp-gke/inputs.tf:182]: done :)

**Review by sunilhonest — APPROVED** [review]

File: inputs.tf [hunk 2]
```diff
@@ -68,6 +68,12 @@ variable "minimum_node_count" {
   description = "Minimum nodes for the node pool. This is the total nodes so for regional deployments it is the total nodes across all zones."
 }
 
+variable "nat_ip_addresses" {
+  type        = list(string)
+  description = "List of IP addresses to use for NAT"
```
Comment (MXfive) [inline inputs.tf:73]: What is the input value here?  The naming and suggestion suggests an actual IP addresses (1.2.3.4), the docs say that a `.self_link` to an `google_compute_address` resource of the correct purpose (`EXTERNAL`) is given.

**Review by MXfive — CHANGES_REQUESTED** [review]

File: inputs.tf (hunk 2 above)
Comment (cyberhck) [reply inputs.tf:73]: you're right, I've changed it to self_link now

**Review by MXfive — APPROVED** [review]

**Pull request merged**